<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <link rel="icon" href="../favicon.ico" />
  <link rel="stylesheet" href="../style.css">
  <title>Shopify CLIを使用したストアへのログインエラー | Techioz</title>
  <style type="text/css">code{white-space: pre;}</style>
  <script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-1637770322917058"
     crossorigin="anonymous"></script>
  <!-- Google tag (gtag.js) -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-DRM1ZRNLDN"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-DRM1ZRNLDN');
  </script>
<script type="application/ld+json">
{
  "@context": "http://schema.org",
  "@type": "Article",
  "headline": "Shopify CLIを使用したストアへのログインエラー",
  "author": {
    "@type": "Person",
    "address": "",
    "email": null,
    "identifier": "masyumaroking",
    "name": "masyumaroking"
  },
  "publisher": {
    "@type": "Person",
    "name": "masyumaroking"
  }
}
</script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
  <header>
    <div class="container">
        <h1>Techioz Blog</h1>
        <nav>
            <ul>
                <li><a href="#">Home</a></li>
                <li><a href="#">Articles</a></li>
                <li><a href="#">About</a></li>
                <li><a href="#">Contact</a></li>
            </ul>
        </nav>
    </div>
  </header>
  <main>
    <div class="container">
        <article>
                      <h2>Shopify
CLIを使用したストアへのログインエラー</h2>
                    <h2 id="概要">概要</h2>
                    <p>ローカル開発を開始するためにストアにログインする際に問題が発生しました。
                    Shopify
                    CLIを使用してこれを試みるのはこれが初めてです。</p>
                    <p>shopify login –store={myStoreName}.myshopify.com
                    を実行すると、次のエラーが発生します。</p>
                    <pre><code>✗ An unexpected error occured.
    To submit an issue include the stack trace.
    To print the stack trace, add the environment variable SHOPIFY_CLI_STACKTRACE=1.

** [Bugsnag] 2022-03-16 14:23:24 +0100: Unable to send information to Bugsnag (https://notify.bugsnag.com), #&lt;OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate)&gt;
** [Bugsnag] 2022-03-16 14:23:24 +0100: [&quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/protocol.rb:46:in `connect_nonblock&#39;&quot;, &quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/protocol.rb:46:in `ssl_socket_connect&#39;&quot;, &quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/http.rb:1038:in `connect&#39;&quot;, &quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/http.rb:970:in `do_start&#39;&quot;, &quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/http.rb:959:in `start&#39;&quot;, &quot;/Users/Tea/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/net/http.rb:1512:in `request&#39;&quot;, &quot;/Users/Tea/.rvm/gems/ruby-3.0.0/gems/bugsnag-6.24.2/lib/bugsnag/delivery/synchronous.rb:51:in `request&#39;&quot;, &quot;/Users/Tea/.rvm/gems/ruby-3.0.0/gems/bugsnag-6.24.2/lib/bugsnag/delivery/synchronous.rb:12:in `deliver&#39;&quot;, &quot;/Users/Tea/.rvm/gems/ruby-3.0.0/gems/bugsnag-6.24.2/lib/bugsnag/delivery/thread_queue.rb:38:in `block in serialize_and_deliver&#39;&quot;, &quot;/Users/Tea/.rvm/gems/ruby-3.0.0/gems/bugsnag-6.24.2/lib/bugsnag/delivery/thread_queue.rb:55:in `block (2 levels) in start_once!&#39;&quot;]
Copy
</code></pre>
                    <p>私が理解しているところによると、これによりブラウザでログインウィンドウが開くはずですが、代わりにこのエラーが発生します。
                    Shopify
                    ログインを行うと、ログインページが表示されます。次にログインしますが、特定のストアに接続していないため、何も変わりません。
                    shopify whoami
                    からログインしていないと言われました。</p>
                    <p>すべてを再インストールしようとしたり、OpenSSL
                    証明書や Shopify CLI
                    情報について調べたり、問題の可能性があると思われる無数のチュートリアルに従ってみたりしましたが、毎回同じエラーが発生します。</p>
                    <p>オペレーティングシステム: macOS</p>
                    <p>シェル: zsh</p>
                    <p>Ruby バージョン (ruby -v): 3.0.3</p>
                    <p>これまでにこの問題に遭遇したことがある人はいますか、または役立つものを教えていただけますか?どうもありがとう。</p>
                    <h2 id="解決策">解決策</h2>
                    <p>同様の問題に遭遇しました。コードで明らかにわかることから、コマンドに
                    = は必要ありません</p>
                    <p>それだけで問題は解決するはずです</p>
        </article>
    </div>
  </main>
  <footer>
    <div class="container">
        <p>&copy; 2024 Techioz. All rights reserved.</p>
    </div>
  </footer>
</body>
</html>